lib/analyzer/static-analyzer.ts

```diff
@@ -70,6 +72,21 @@ export async function analyze(
     staticAnalysisActions.push(getDpkgPackageFileContentAction);
   }
 
+  const checkForGlobs =
+    options &&
+    options.globsToFind &&
+    options.globsToFind.include &&
+    Array.isArray(options.globsToFind.include) &&
+    options.globsToFind.include.length > 0;
+  if (checkForGlobs) {
+    staticAnalysisActions.push(
+      filePatternStatic.generateExtractAction(
+        options.globsToFind.include,
+        options.globsToFind.exclude,
```

Comment:
Should we also assert that `exclude` is a `string[]`? Looks like we pass it all the way to `generatePathMatcher` and assume it's a `string[]`

Reply:
I'm not sure.
I mean, yes, we should, but I don't want this area to be responsible for input validation.
the plugin gets plenty of options, but I don't think it asserts their correctness at any point... it may be a valid approach ("don't misuse the options, or prepare to catch your own exceptions") but if we're opting to protect the library's caller then I want to invest less here and more in a more centralised module to check all options once.